x/evm/keeper/keeper.go

```diff
@@ -345,12 +345,7 @@ func (k Keeper) getBaseFee(ctx sdk.Context, london bool) *big.Int {
 
 // GetMinGasMultiplier returns the MinGasMultiplier param from the fee market module
 func (k Keeper) GetMinGasMultiplier(ctx sdk.Context) sdk.Dec {
-	fmkParmas := k.feeMarketKeeper.GetParams(ctx)
-	if fmkParmas.MinGasMultiplier.IsNil() {
-		// in case we are executing eth_call on a legacy block, returns a zero value.
-		return sdk.ZeroDec()
-	}
-	return fmkParmas.MinGasMultiplier
```

Comment:
@facs95 it's `debug_trace*` API, there is a fix here for legacy blocks. But that is not enough. There are 2 nil-able params are `MinGasMultiplier`and `MinGasPrice`.

[`CalculateBaseFee`](https://github.com/evmos/evmos/blob/b2b08b6e0994c300e48d449823d009aa33677479/x/feemarket/keeper/eip1559.go#L18C17-L18C33) (EIP-1559) access `MinGasPrice` params thus panic when accessing from legacy blocks.